### PR TITLE
feat(suite): remove RBF option from the SendForm, make all Transactions with RBF enabled by default

### LINKED ISSUE #13160: Full RBF (step 1/2)
- [x] All new transactions has RBF `ON` (remove UI to enable/disable it)
- ~[ ] Leave `[Finalize Transaction]`~ Its already gone: https://github.com/trezor/trezor-suite/pull/11730/files

Followup https://github.com/trezor/trezor-suite/issues/13159

### PR DESCRIPTION
**Pull request by peter-sanderson**
Resolves https://github.com/trezor/trezor-suite/issues/13160

### CODE REVIEW

File: packages/suite/src/views/wallet/send/components/Options/BitcoinOptions/BitcoinOptions.tsx [hunk 1]
```diff
@@ -115,33 +114,6 @@ export const BitcoinOptions = () => {
                             </StyledButton>
                         </Tooltip>
                     )}
-
-                    {isFeatureFlagEnabled('RBF') &&
-                        network.features?.includes('rbf') &&
-                        !locktimeEnabled && (
-                            <Tooltip
-                                addon={
-                                    <OpenGuideFromTooltip id="/3_send-and-receive/5_replace-by-fee-rbf.md" />
```
Comment (komret) [inline packages/suite/src/views/wallet/send/components/Options/BitcoinOptions/BitcoinOptions.tsx:124]: Reminder: remove the article from Guide

File: suite-common/wallet-core/src/send/sendFormBitcoinThunks.ts [hunk 2]
```diff
@@ -58,11 +58,8 @@ export const composeBitcoinSendFormTransactionThunk = createThunk(
             });
         }
 
-        let sequence; // Must be undefined for final transaction.
-        if (formValues.options.includes('bitcoinRBF')) {
-            // RBF is set, add sequence to inputs
-            sequence = BTC_RBF_SEQUENCE;
-        } else if (formValues.bitcoinLockTime) {
+        let sequence = BTC_RBF_SEQUENCE;
```
Comment (peter-sanderson) [inline suite-common/wallet-core/src/send/sendFormBitcoinThunks.ts]: Is this safe?

File: suite-common/wallet-core/src/send/sendFormThunks.ts [hunk 3]
```diff
@@ -471,7 +471,7 @@ export const enhancePrecomposedTransactionThunk = createThunk<
         };
 
         if (!isCardanoTx(selectedAccount, enhancedPrecomposedTransaction)) {
-            enhancedPrecomposedTransaction.rbf = formValues.options.includes('bitcoinRBF');
+            enhancedPrecomposedTransaction.rbf = true; // RBF is enabled for all new transaction
```
Comment (peter-sanderson) [inline suite-common/wallet-core/src/send/sendFormThunks.ts]: Can we somehow get rid of this completely?

**Review by komret — APPROVED** [review]
🧹

File: packages/suite/src/views/wallet/send/components/Options/BitcoinOptions/BitcoinOptions.tsx (hunk 1 above)
Comment (komret) [reply packages/suite/src/views/wallet/send/components/Options/BitcoinOptions/BitcoinOptions.tsx:124]: [Knowledge Base](https://trezor.io/learn/a/replace-by-fee-rbf-bitcoin) must also be updated, I notified @b-irving.

**Pull request merged**